framework/core/less/common/Modal.less

```diff
@@ -45,41 +48,7 @@
   -webkit-overflow-scrolling: touch;
   overflow-x: hidden;
   overflow-y: auto;
-
-  // This mess of data attribute selectors ensures that the backdrop correctly
-  // transitions when modals are stacked.
-  //
-  // When a modal is stacked on top of another, the original backdrop is
-  // immediately hidden, and the new backdrop is immediately shown. Same with
-  // removing a stacked modal: the topmost backdrop is immediately hidden,
-  // and the next backdrop is shown.
-  //
-  // This prevents flashes of backdrops being shown/hidden as multiple
-  // transitions occur simultaneously.
-  &[data-visibility-state="entered-underneath"] {
-    .backdrop {
-      opacity: 0;
-      transition: none;
-    }
-
-    + .ModalManager[data-visibility-state="entering"] {
-      .backdrop {
-        transition: none;
-      }
-    }
-  }
-
-  &[data-visibility-state="entered"] {
-    .backdrop {
-      transition: none;
-    }
-
-    + .ModalManager[data-visibility-state="exiting"] {
-      .backdrop {
-        transition: none;
-      }
-    }
-  }
+  pointer-events: none;
```

Comment:
so the scrollbar shows up correctly but with `pointer-events: none` I can't scroll. But removing this also means you can't click outside to close 🤔